packages/story-editor/src/components/floatingMenu/elements/trim.js

```diff
@@ -44,12 +45,18 @@ function Trim() {
     ? __('Trimming video', 'web-stories')
     : __('Trim video', 'web-stories');
 
+  const handleClick = (e) => {
+    trackEvent('floating_menu', {
+      name: 'set_trim',
+    });
```

Comment:
and here?

Reply:
all 1 element type option so it's excess to track. unnecessary :)